galeracluster/pasture/source/index.rst

```diff
@@ -45,7 +45,7 @@ Other standard MySQL functionality, if it is not related to Galera, is not
 covered by this manual, rather the reader should refer to the following official 
 documentations for each product:
 
-* `MySQL Reference Manual <http://dev.mysql.com/doc/refman/5.5/en/index.html>`_
+* `MySQL Reference Manual <https://dev.mysql.com/doc/refman/8.4/en/>`_
```

Comment:
Please also add a link to the 8.0 documentation. This should be in general - 8.0 + 8.4

Reply:
Fixed